ui/desktop/app/styles/app.scss

```diff
@@ -705,3 +705,13 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     }
   }
 }
+
+.sessions {
```

Comment:
Are these styles applicable to all search and filtering implementations? If so, Perhaps we should be generalizing these styles rather than adding resource specific ones that will be duplicates

Reply:
good point, we should generalize them!